hack/generate-manifest.sh

```diff
@@ -323,12 +324,14 @@ EOF
 fi
 
 if [ "$MODE" == "dev" ]; then
-    if [[ -z "$IMG_NAME" ]]; then
+    if [[ -z "$AGENT_IMG_NAME" ]] || [[ -z "$CONTROLLER_IMG_NAME" ]]; then
```

Comment:
If `AGENT_IMG_NAME` is set and `CONTROLLER_IMG_NAME` is unset, both `controllerImage` and `agentImage` repository will be hardcoded. Moreover, in this case, if COVERAGE is false, no value will be added to HELM_VALUES. Not sure if this meets you expectation?

Reply:
> If `AGENT_IMG_NAME` is set and `CONTROLLER_IMG_NAME` is unset, both `controllerImage` and `agentImage` repository will be hardcoded.

Please address this comment.

Reply:
yes, in dev mode we will be hardcoding both.

Reply:
to me it feels that both images should be handled separately:
```
if [[ -z "$AGENT_IMG_NAME" ]]; then
    # ...
fi
if [[ -z "$CONTROLLER_IMG_NAME" ]]; then
    # ...
fi
```